.github/workflows/osdk_test.yml

```diff
@@ -50,5 +50,5 @@ jobs:
         # So the RUSTUP_HOME needs to be set here. 
         # This only breaks when we invoke Cargo in the integration test of OSDK 
         # since the OSDK toolchain is not nightly.
-        run: cd osdk && cargo build && RUSTUP_HOME=/root/.rustup cargo test
+        run: cd osdk && RUSTUP_HOME=/root/.rustup cargo +stable build && RUSTUP_HOME=/root/.rustup cargo test
```

Comment:
Since `osdk/rust-toolchain.toml` is deleted, I think it's also worthwhile to specify `+stable` in the root `Makefile` when installing the OSDK.

Reply:
> Another reason is that when I searching several cargo subcommands, like [cargo-expand](https://github.com/dtolnay/cargo-expand), [cargo-deny](https://github.com/EmbarkStudios/cargo-deny), [cargo-edit](https://github.com/killercup/cargo-edit), they all not specify the channel as stable.

I think @StevenJiang1110 has made a good point. Since OSDK is for kernel development and these kernels are required to use nightly Rust anyway, what is the point of insisting on using stable Rust for building OSDK?

Reply:
Well, if we want to use stable channel in Makefile, it seems that we need to add `RUSTUP_HOME=/root/.rustup` for each `make run` in `kernel_test.yml` for the same reason.

But it may not bring actual benefits, since our `osdk_test.yml` already ensures the OSDK can be built with stable channel.